orangecontrib/text/widgets/owconcordance.py

```diff
@@ -0,0 +1,263 @@
+from typing import Optional
```

Comment:
This is new in Python 3.5. To make it also work with Python 3.4 please add `typing` to requirements with a comment that it is required on py34.